pom.xml

```diff
@@ -29,13 +29,13 @@
 
     <modules>
         <module>core</module>
-        <module>ams</module>
+        <module>hive</module>
         <module>optimizer</module>
         <module>flink</module>
         <module>spark</module>
-        <module>dist</module>
         <module>trino</module>
-        <module>hive</module>
+        <module>ams</module>
+        <module>dist</module>
     </modules>
```

Comment:
why change the order of modules?

Reply:
because package ams and dist depend on other module, the old order of modules maybe cause ams and dist package with old dependency